cmd/query/main_test.go

```diff
@@ -108,6 +108,26 @@ func TestSliceErrorHTTPResponse(t *testing.T) {
 			},
 			testSliceRequest{},
 		},
+		sliceTest{
+			baseTest{
+				name:           "Invalid json GET request",
+				method:         http.MethodGet,
+				jsonRequest:    "nil",
+				expectedStatus: http.StatusBadRequest,
+				expectedError:  "The specified endpoint requires a 'query' parameter ",
+			},
+			testSliceRequest{},
+		},
+		sliceTest{
+			baseTest{
+				name:           "Invalid json GET request",
```

Comment:
This test is actually the same as original one. What's the point in having two? So I think you can either:
1) Exchange original test with your test. It checks just presence of the error so it is fine. Original message is not really important
2) Add the full string to the test, thus combining both tests (would be fine, but the string is loong :smile: )
3) add to baseTest expectedErrorRegex and if it is present in the testcase compare with require.Regex. Then the string from 2 becomes shorter. Don't really see much value in this particular case, but can be as an option.

I think I would be fine with just option 1, but see if you prefer something else

Reply:
I'm also agree with option one. So will change the test case